src/docdb/tree/DocDBStoredProcedureTreeItem.ts

```diff
@@ -44,7 +49,11 @@ export class DocDBStoredProcedureTreeItem extends AzureTreeItem<IDocDBTreeRoot>
     }
 
     public async getFileContent(): Promise<string> {
-        return this.procedure.body;
+        const clonedProcedure: {} = { ...this.procedure?.body?.toString };
```

Comment:
How come the behavior changed here?

Reply:
![image](https://user-images.githubusercontent.com/17537419/99455385-9e60eb00-28dc-11eb-8adb-9c3c10bce20d.png)
Actually, modifications to the string that are made here are not necessary , but it requires a small fix as `body` returns `string | ((...inputs: any[]) => void`.